src/environments/environment.deploy.ts

```diff
@@ -2,5 +2,6 @@ export const environment = {
   production: false,
   VTM_CLIENT_ID: '',
   VTM_AUTHORITY_ID: '',
-  VTM_REDIRECT_URI: '/'
+  VTM_REDIRECT_URI: 'http://localhost:4200',
+  VTM_API_URI: 'http://localhost:3005'
```

Comment:
This might want renaming to VTM_TECH_RECORD_API

Reply:
Looks like Tech and Test records are paths under the same API in AWS, though I may be reading that incorrectly, can we discuss it?

Reply:
Aha no you are right, we just need one then